public/locales/zh-TW/dashboard.json

```
@@ -844,6 +844,7 @@
     "cos": "騰訊雲 COS",
     "onedrive": "OneDrive",
     "s3": "S3 相容",
+    "ks3": "KS3 相容",
```

Same as public/locales/en-US/dashboard.json.
```suggestion
    "ks3": "金山雲 KS3",
```